bindings/Cargo.toml

```diff
@@ -35,25 +35,25 @@ resolver = "2"
   serde                   = "1"
   serde-wasm-bindgen      = "0.6.5"
   serde_json              = "1.0.140"
-  swc_atoms               = "7.0.0"
-  swc_cli_impl            = "40.0.0"
-  swc_common              = "14.0.3"
-  swc_compiler_base       = "34.0.0"
-  swc_config              = "3.1.1"
-  swc_core                = "40.0.0"
-  swc_css_ast             = "14.0.0"
-  swc_css_codegen         = "14.0.0"
-  swc_css_minifier        = "14.0.0"
-  swc_css_parser          = "14.0.0"
-  swc_ecma_react_compiler = "8.0.0"
-  swc_error_reporters     = "16.0.1"
-  swc_html                = "26.0.0"
-  swc_html_ast            = "14.0.0"
-  swc_html_minifier       = "32.0.0"
-  swc_malloc              = "1.2.3"
-  swc_nodejs_common       = "1.0.2"
+  swc_atoms               = { path = "../crates/swc_atoms" }
```

Comment:
Can you revert these changes? This breaks nodejs

Reply:
I can't upgrade NAPI-RS independently in `bindings/*` because the `crates/*` also depend on `napi` and `napi-derive`.